src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3828,18 +3828,19 @@
      * <p>For example:</p>
      *
      * <pre>
-     *     { 1, 2 }
+     * &#64;Annotation({1, 2})
      * </pre>
      *
      * <p>parses as:</p>
      *
      * <pre>
-     * +--ANNOTATION_ARRAY_INIT ({)
-     *     |
-     *     +--NUM_INT (1)
-     *     +--COMMA (,)
-     *     +--NUM_INT (2)
-     *     +--RCURLY (})
+     * ANNOTATION_ARRAY_INIT -&gt; {
```

Comment:
since our example starts with annotation, I think our AST tree should also start with `ANNOTATION -> ANNOTATION` node

it will be easier to understand for users, what do you think?

Reply:
Yes you are right. I will add this.